lib/filter/enemy.js

```diff
@@ -99,7 +99,31 @@ exports.update = function (json) {
       }
     }
     */
-
+   
+		console.log( 'Enemy Drops:');
+		var drops = 0;
+		for (enemy in json[round].enemy) {
+			if (json[round].enemy[enemy].children[part].drop_item_list.length > 0) {
+				drops = 1;
+				for (var item in json[round].enemy[enemy].children[part].drop_item_list) {
```

Comment:
I did some digging on this - it's really interesting. As was already in the code, there is actually 2 different sets of drops:

Round drops, which come in the first entity, which always tend to happen at the end of a round - such as potions (going from ID 21, 22, 31, 32, maybe more? But I've only ever seen 21, 22, and 31) and Materia (for RM! Got one yesterday, that was a pleasant surprise in the output).

Then enemy drops, which always appear as a subset of an enemy char. This is usually for bosses, but is actually for regular enemies as well (certain enemies will be flagged to drop certain armor/weapons/orbs). Not so important that we want to say which enemy will drop which (because they all will die :]), so I just throw it in the round rather than per enemy.

If I move the code though, it will be per enemy. It should work, I think. It'll just be a little sloppy? I dont know what it'll look like since it's technically inserting into a table (or maybe it's not, i dunno). 

We'll give it a whirl tonight.